fullscreen.html

```diff
@@ -7,9 +7,9 @@
     <link rel="shortcut icon" type="image/ico" href="images/favicon.ico"/>
     <script type="text/javascript" src="inverse-analytics.js"></script>
     <noscript><p><img src="//stats.opkode.com/piwik.php?idsite=5" style="border:0;" alt="" /></p></noscript>
-    <link type="text/css" rel="stylesheet" media="screen" href="https://cdn.conversejs.org/4.2.0/css/converse.min.css" />
+    <link type="text/css" rel="stylesheet" media="screen" href="dist/converse.min.css" />
     <script src="https://cdn.conversejs.org/3rdparty/libsignal-protocol.min.js"></script>
-    <script src="https://cdn.conversejs.org/4.2.0/dist/converse.min.js"></script>
+    <script src="dist/converse.min.js"></script>
 </head>
 <body class="converse-fullscreen">
 <div id="conversejs-bg"></div>
```

Comment:
Please don't include this change in the PR.

If you want to test your changes locally, you can use the [dev.html][https://github.com/conversejs/converse.js/blob/master/dev.html) instead of this one.

Reply:
Thank you, I have made the changes you asked for :D